src/utils/variableType.js

```diff
@@ -7,10 +8,28 @@ import {isFunction} from './functionType';
  * @return {Boolean}
  */
 export function isUpdate(node) {
-  return node.type === 'UpdateExpression' && node.argument.type === 'Identifier';
+  return node.type === 'UpdateExpression';
 }
 
 /**
+ * True when node is an assignment with the variable on the LHS (like x= foo).
+ *
+ * @param {Object} node
+ * @param {Object} parent Immediate parent node (to determine context)
+ * @return {Boolean}
+ */
+export function isOnLHSAssignment(node, parent) {
```

Comment:
It took me a while to figure out what `LHS` means. Left Hand Side, I guess. A better name would IMHO be `isAssignedTo`.

On a more generic note, I think this function doesn't really belong in here (as it doesn't deal specifically with variable assignments, but with all kind of assignments) it can just as well live inside forEach transform itself.